pkg/oci/load_balancer_security_lists_test.go

```diff
@@ -382,13 +415,90 @@ func TestGetLoadBalancerEgressRules(t *testing.T) {
 				{CidrBlock: common.String("1")},
 				{CidrBlock: common.String("2")},
 			},
-			port:     80,
-			services: []*v1.Service{},
+			actualPort:  80,
+			desiredPort: 80,
+			services:    []*v1.Service{},
 			expected: []core.EgressSecurityRule{
 				makeEgressSecurityRule("existing", 9000),
 				makeEgressSecurityRule("1", 80),
 				makeEgressSecurityRule("2", 80),
 			},
+		}, {
```

Comment:
Unit test for service port mutation. I chose to put more realistic strings in. IMHO it makes these tests easier to read/understand, especially if we added further input rules in. I can make it like the other though.